playwright.config.ts

```diff
@@ -28,7 +29,53 @@ export default defineConfig({
   /* Opt out of parallel tests on CI. */
   workers: process.env.CI ? 4 : 6,
   /* Reporter to use. See https://playwright.dev/docs/test-reporters */
-  reporter: 'html',
+  reporter: [
+    [
+      './node_modules/playwright-slack-report/dist/src/SlackReporter.js',
+      {
+        channels: ['health_check'], // provide one or more Slack channels
+        sendResults: 'always', // "always" , "on-failure", "off"
+        maxNumberOfFailuresToShow: 100,
+        meta: [
+          {
+            key: `Tests launched 🎬`,
+            value: `${process.env.CI ? 'Github Actions 🐌' : 'Locally'}`,
+          },
+          {
+            key: `Environment 👷🏻‍♂️`,
+            value: `${process.env.ENVIRONMENT?.toUpperCase()}`,
```

Comment:
Maybe
```suggestion
            value: process.env.ENVIRONMENT?.toUpperCase() ?? '',
```
will be better?

Reply:
actually this variable can not be undefined at the report sending step.check for process.env.ENVIRONMENT is added just to make TS calmer when calling toUpperCase() method. It is ok if we fall if env is somehow not set as all tests use this variable behind the scenes. So no, it makes no sense